nautobot/dcim/navigation.py

```diff
@@ -319,20 +319,12 @@
                         ),
                     ),
                     NavMenuItem(
-                        link="dcim:controllerdevicegroup_list",
-                        name="Controller Device Groups",
+                        link="dcim:controllermanageddevicegroup_list",
+                        name="Controller Managed Device Groups",
```

Comment:
I think we'd decided that in the menu this would be just `Managed Device Groups`?

Reply:
Tests fails on this.

Reply:
https://github.com/nautobot/nautobot/actions/runs/8388114257/job/22971812947#step:7:166

Reply:
There's an existing pattern in `nautobot/core/tests/test_navigations.py` for case-by-case overrides of the default logic there, just need to add this case.

Reply:
Thanks, fixed